src/commands/deployWorkspaceProject/DeployWorkspaceProjectSaveSettingsStep.ts

```diff
@@ -27,4 +36,26 @@ export class DeployWorkspaceProjectSaveSettingsStep extends AzureWizardExecuteSt
     public shouldExecute(context: DeployWorkspaceProjectContext): boolean {
         return !!context.shouldSaveDeploySettings;
     }
+
+    protected initSuccessOutput(): ExecuteActivityOutput {
+        return {
+            item: new GenericTreeItem(undefined, {
+                contextValue: createActivityChildContext(['deployWorkspaceProjectSaveSettingsStep', activitySuccessContext]),
+                label: saveSettingsLabel,
+                iconPath: activitySuccessIcon
+            }),
+            output: localize('savedSettingsSuccess', 'Saved deployment settings to workspace: "{0}".', relativeSettingsFilePath)
```

Comment:
Weird that this one has a colon, but the other resources output messages do not.